WeatherDispatcher/src/main/java/com/example/WeatherDispatcher/utils/TemperatureUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.example.WeatherDispatcher.utils;
+
+public class TemperatureUtil {
+
+    public static boolean checkDiff(Double num1, Double num2, Integer threshold) {
```

Comment:
how checkdiff is related to Temperature???

Reply:
done